base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2538,7 +2553,14 @@ impl BlockchainBackend for LMDBDatabase {
         // We do this to ensure backwards compatibility on older exising dbs that did not store a reason
         let exist = lmdb_exists(&txn, &self.bad_blocks, block_hash.deref())?;
         match lmdb_get::<_, (u64, String)>(&txn, &self.bad_blocks, block_hash.deref()) {
-            Ok(Some((_height, reason))) => Ok((true, reason)),
+            Ok(Some((height, reason))) => {
+                debug!(
```

Comment:
```suggestion
                trace!(
```
Its going to double log if you have a bad block, we can make this now trace